sdk/ai/Azure.AI.Projects/samples/Sample20_MemoryStore.md

```diff
@@ -0,0 +1,188 @@
+# Sample of using `MemoryStore` in Azure.AI.Projects.OpenAI.
+
+In this example we will demonstrate how to create, update, list and delete memory stores and scopes inside them.
+
+1. First, we need to create project client and read the environment variables, which will be used in the next steps.
+
+```C# Snippet:Sample_MemoryStore
+var projectEndpoint = System.Environment.GetEnvironmentVariable("PROJECT_ENDPOINT");
+var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
+var embeddingDeploymentName = System.Environment.GetEnvironmentVariable("EMBEDDING_MODEL_DEPLOYMENT_NAME");
+AIProjectClient projectClient = new(new Uri(projectEndpoint), new DefaultAzureCredential());
+```
+
+2. Use the client to create the `MemoryStore`. Memory store requires two models, one for embedding and another for search.
```

Comment:
"for search"? Is this the right term? Maybe there is a better term on learn.microsoft docs. They call their model "chat model" (because they use chat completions internally to look up relevant memory? not sure why)